packages/components/src/modal/index.js

```diff
@@ -162,6 +157,12 @@ function Modal( props, forwardedRef ) {
 							'has-scrolled-content': hasScrolledContent,
 						} ) }
 						role="document"
+						tabIndex="0"
+						ref={ useMergeRefs( [
+							constrainedTabbingRef,
+							focusReturnRef,
+							focusOnMountRef,
+						] ) }
```

Comment:
I think that moving this ref prop broke constraining tabbing within the modal. Testing the 'Keyboard shortcuts' modal in the 'Options' menu, I'm not able to tab away from the Close button. Was this change intentional?

Reply:
@afercia I could never tab within this modal. I am not sure how you were able to all things considered. If the modal wrapping element has tabindex of -1, that would mean the content inside would need at least tabindex of 0 to be accessible via Tab key.

Reply:
Also, the failing E2E tests look unrelated. I know a couple of these fail all the time. I will try another re-run. Some of these tests are just pesky and annoying since you do not know if your PR is making it fail or if the tests are just having a bad day.

Reply:
@alexstine regarding tabbing within the modal: the PR adds tabindex=0 to the element with role=document so that there are now two focusable elements within the modal:
- the Close button
- the element with tabindex=0, which is the scrollable div

I'm not able to tab to the scrollable div. Maybe for testing you could try to add an aria-label to the scrollable div and see if you can tab to it. When I move back the ref prop to its original place, tabbing to the scrollable div works. Kind reminder that this should be tested with Chrome.

Reply:
@afercia Here is what I did.

Checkout trunk:
1. Open Google Chrome and navigate to a new post.
2. Select Options > Keyboard Shortcuts.
3. Press tab to land on the Close dialog button.
4. Press tab again, nothing is announced.

Is this a visual thing that tab can move you in to the scrolling div? I sure do not hear anything. Between trunk and my branch, nothing seems different. This leads me to believe this is purely visual.

Thanks.